src/main/java/db/structure/Column.java

```diff
@@ -48,10 +79,27 @@ private void readObject(java.io.ObjectInputStream in) throws IOException, ClassN
 	 * @param dataType
 	 */
 	public Column(String name, DataType dataType) {
+		this(name, dataType, true, true);
+	}
+	
+	public Column(String name, DataType dataType, boolean argKeepDataInMemory) {
+		this(name, dataType, argKeepDataInMemory, false);
+	}
+	/** 
+	 *  @param name
+	 *  @param dataType
+	 *  @param argKeepDataInMemory  garder en mémoire les données ou juste les écrire sur le disque
+	 *  @param argWriteDataOnDisk   écrire la donnée sur le disque
+	 */
+	public Column(String name, DataType dataType, boolean argKeepDataInMemory, boolean argWriteDataOnDisk) {
```

Comment:
alors le Keep data in memory officiellement j'aime pas trop. Du tout.

Pour moi ça correspond à la liste des indexes

Reply:
C'est pas le cas pour moi : tu peux vouloir avoir de la donnée en mémoire alors qu'elle n'est pas indexée !!

Reply:
hmmmm, oui c'est vrai que je n'avais pas tout à fait compris à quoi ça correspondait à ce moment de la review. ouais disons qu'on laisse comme ça